package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "chainlink",
-  "version": "2.23.2",
+  "version": "2.22.1",
```

Comment:
Why change this?

Reply:
it was only for the image build, let me drop this commit.